lib/src/base-styles/_nx-globals.scss

```diff
@@ -15,8 +15,11 @@ $-nx-scrollable-default-height: 400px;
 .nx-text-link {
   @include nx-container-helpers.container-horizontal;
 
+  border: 1px solid transparent;
   color: var(--nx-color-link);
   cursor: pointer;
+  margin: 0 calc(0px - #{var(--nx-spacing-base)});
```

Comment:
Why is there interpolation syntax here?

Reply:
> Why is there interpolation syntax here?

Weird muscle memory I think: https://github.com/sonatype/sonatype-react-shared-components/pull/502/commits/f306a7ab262b132447388637a981726832f6b0ca

Reply:
> > Fixed here and confirmed by measuring screenshots: [f306a7a](https://github.com/sonatype/sonatype-react-shared-components/commit/f306a7ab262b132447388637a981726832f6b0ca)
> 
> That has too little margin. Margin should cancel out padding + border

In the commit above padding is 4px - 1px = 3px + 1px border = 4px total. The margin is 4px.